### PR TITLE
[Feature] Implement Card

### LINKED ISSUE #313: [Feature] Card
**Is your feature request related to a problem? Please describe.**
nope

**Describe the solution you'd like**
a Card component : Simple rectangular container.
- title, subtitle, coverImage support
- loading, disabled, raised status support
- styling

Screen Shot 2020-08-22 at 11 38 23 AM

[go figma workspace to get more details](https://www.figma.com/file/f7LwH54h08Q6nsBNs7bCvd/DOOBOO-UI-CARD?node-id=0%3A1)

implementing it [here](https://github.com/doobooUI-card-team/dooboo-ui)

### PR DESCRIPTION
**Pull request by greatSumini**
## Description

a Card component : Simple rectangular container with title, subtitle, cover image.

Card

## Related Issues

resolve #313 

## Tests

I added the following tests:

- Simple rendering snapshot tests of the props of Card

## Checklist

Before you create this PR confirms that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide](https://github.com/dooboolab/dooboo-ui/blob/master/CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
- [x] Run `yarn test` or `yarn test -u` if you need to update snapshot.
- [x] Run `yarn lint`
- [x] I am willing to follow-up on review comments in a timely manner.

### CODE REVIEW

File: main/Card/index.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  ViewStyle,
+} from 'react-native';
+import React, { FC, ReactNode, ReactNodeArray } from 'react';
+
+import { FlattenSimpleInterpolation } from 'styled-components';
+import styled from 'styled-components/native';
+
+import styles from './styles';
+
+interface IContainerProps {
```
Comment (JeffGuKang) [inline main/Card/index.tsx]: Just use `ContainerProps` without prefix I.

File: main/Card/index.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  outlined?: boolean;
+}
+
+const Container = styled.View<IContainerProps>`
+  flex-direction: column;
+  align-items: flex-start;
+  background-color: #fff;
+  width: 256px;
+  ${({ outlined, raised }): FlattenSimpleInterpolation =>
+    outlined ? styles.border : raised ? styles.raisedShadow : styles.shadow}
```
Comment (JeffGuKang) [inline main/Card/index.tsx:26]: It causes warning as `Node of type rule not supported as an inline style` on iOS and Android. 
Find another way.

File: stories/dooboo-ui/Card.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,155 @@
...
+export const toStorybook1 = (): ReactElement => <Style />;
+
+toStorybook.story = {
+  name: 'default',
+};
+
+toStorybook1.story = {
+  name: 'Style',
+};
+/**
```
Comment (JeffGuKang) [inline stories/dooboo-ui/Card.stories.tsx:142]: break the line.

**Review by JeffGuKang — CHANGES_REQUESTED** [review]

File: stories/dooboo-ui/Card.stories.tsx (hunk 3 above)
Comment (sk97kang) [reply stories/dooboo-ui/Card.stories.tsx:142]: I'd like you to explain a little more.

Comment (JeffGuKang) [reply stories/dooboo-ui/Card.stories.tsx:142]: It needs a line break between 140 and 141 line number.

Comment (sk97kang) [reply stories/dooboo-ui/Card.stories.tsx:142]: Thank you! I understood and corrected.

**Comment by JeffGuKang** [comment]
Good feature. There are some minor stuffs waiting your touch. It will be merged after fix them.

File: main/Card/index.tsx (hunk 1 above)
Comment (JeffGuKang) [reply main/Card/index.tsx]: Refs: https://github.com/microsoft/TypeScript-Handbook/issues/121#issuecomment-412810455

File: main/Card/index.tsx (hunk 2 above)
Comment (greatSumini) [reply main/Card/index.tsx:26]: The cause was shadow-offset!
fixed in 1842cbf

File: main/Card/index.tsx (hunk 1 above)
Comment (greatSumini) [reply main/Card/index.tsx]: thk u for comment + refs! 😆 
resolved in 2b761a7

**Comment by greatSumini** [comment]
> Good feature. There are some minor stuffs waiting your touch. It will be merged after fix them.

fixed all of them! 😄   @JeffGuKang 

**ps**
again, CI/CD mark is not visible on my PR T.T
it seems like `circle ci` hates me 😂

**Comment by JeffGuKang** [comment]
> again, CI/CD mark is not visible on my PR T.T
> it seems like `circle ci` hates me 😂

I cannot understand why CI is not working on only your commit. 
Anyway, this PR looks good to me!! Great work!

**Pull request merged**